src/test/objectstore/store_test.cc

```diff
@@ -3296,7 +3296,6 @@ TEST(EXT4StoreTest, _detect_fs) {
   // without user_xattr, ext4 fails
   //
   {
-    g_ceph_context->_conf->set_val("filestore_xattr_use_omap", "true");
```

Comment:
@dachary what do you think about the change regarding to this test?

shall we just remove it?